x2gbfs/x2gbfs.py

```diff
@@ -71,8 +71,8 @@ def build_extractor(provider: str, feed_config: Dict[str, Any]) -> BaseProvider:
         return MoqoProvider(feed_config)
     if provider == 'lara_to_go':
         return LaraToGoProvider(feed_config)
-    if provider in ['noi']:
-        return NoiProvider(feed_config)
+    if provider in ['opendatahub']:
```

Comment:
As explained above, you might want to list the car sharing providers instead of the API provider

Reply:
I renamed it to `alpsgo`.